mix.exs

```diff
@@ -38,7 +38,9 @@ defmodule ExGram.Mixfile do
     [
       maintainers: ["Miguel Garcia / Rock Neurotiko"],
       licenses: ["Beerware"],
-      links: %{"GitHub" => "https://github.com/rockneurotiko/ex_gram"}
+      links: %{"GitHub" => "https://github.com/rockneurotiko/ex_gram"},
+      files:
+        ~w(lib priv .formatter.exs mix.exs README* readme* LICENSE* license* CHANGELOG* changelog* src templates)
```

Comment:
You don't need all that, with this should be enough:

```suggestion
        ~w(lib templates mix.exs README.md)
```

Reply:
Nice, I really didn't know what to put so I just took all the default stuff 😂